src/OrleansAWSUtils/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Orleans;
+using Orleans.Runtime.Configuration;
+using Orleans.Runtime.MembershipService;
+using OrleansAWSUtils.Configuration;
+
+namespace OrleansAWSUtils
```

Comment:
I'd use Microsoft.Orleans.Hosting as the namespace for all these extension methods. Thoughts? /cc @ReubenBond @attilah

Reply:
It make sense since it is extension method for Microsoft.Orleans.Hostting.

Reply:
@jdom yes, putting them where they make more sense:
- Either where they're adding functionality: Hosting
- Or where extended interface/class besides: ```IServiceCollection```